spec/models/curator/filestreams/video_spec.rb

```diff
@@ -8,10 +8,22 @@
 
   it_behaves_like 'file_set'
 
-  describe 'Associations' do
+  describe 'Video Associations' do
     it { is_expected.to belong_to(:file_set_of).
                         inverse_of(:video_file_sets).
                         class_name('Curator::DigitalObject').
                         required }
+
```

Comment:
Is there a way to DRY out the `describe 'File Attachments` blocks in the `spec/models/curator/filestreams/*_spec.rb` files using shared examples? You might be able to set a generic `let!(:file_attachments)` helper that could be called in the shared examples?

(Good idea adding these in, btw.)